ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -768,4 +770,36 @@ private void executeDdlCommand(final DdlCommand ddlCommand) {
       throw new KsqlException(ddlCommandResult.getMessage());
     }
   }
+
+  private void checkPersistentQueryCapacity(
+      final List<PreparedStatement> parsedStatements, final String queriesString) {
+
+    final long numPersistentQueries = parsedStatements.stream().filter(parsedStatement -> {
+      final Statement statement = parsedStatement.getStatement();
```

Comment:
Humm.... 

This increases our chance of writing commands to the command topic that exceed our quota.  Can we not just parse the script?  We already parse non-script statements before  writing them to the command topic. I actually think it would be beneficial to parse the script too. Means we don't write dodgy scripts to the command topic.

Of course, I'd do that in a follow up PR, not in here!

Reply:
Did you have thoughts on this?  Worth raising a github issue for yourself to track and referencing here if you agree.

Reply:
Turns out [an issue](https://github.com/confluentinc/ksql/issues/1230) already exists for this. Just assigned myself.